src/components/Footer/Footer.js

```diff
@@ -1,5 +1,5 @@
-import * as React from 'react';
 import PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed. This is not a substantive issue as it doesn't affect the functionality of the code.
```diff
- import * as React from 'react';
- import PropTypes from 'prop-types';
+ import PropTypes from 'prop-types';
+ import * as React from 'react';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->